src/integration-test/java/com/commercetools/sync/integration/externalsource/products/ProductSyncWithPricesIT.java

```diff
@@ -586,6 +591,220 @@ void withMixedCasesOfPriceMatches_ShouldBuildActions() {
         assertPricesAreEqual(productProjection.getMasterVariant().getPrices(), newPricesWithReferenceIds);
     }
 
+    @Test
+    void withNonEmptyPriceCustomFieldSet_ForEmptyPriceCustomField_ShouldCreateCustomFieldValue() {
```

Comment:
Please follow the test naming convention: 

`methodUnderTest_testCondition_expectedResult`, in this case:


`sync_WithEmptyOldCustomFieldAndNonEmptyNewField_ShouldCreateCustomFieldValue`

Reply:
cc77a94165412b5c689e3170647af8e9a5c2eb55

Reply:
thanks. Is there a case for sync_WithNullOldCustomFieldAndEmptyNewField_ShouldSetCustomField

Reply:
9f1bcf1